Testscripts/Linux/perf_mariadb.sh

```diff
@@ -0,0 +1,292 @@
+#!/bin/bash
+
+#######################################################################
+#
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache License.
+#
+#######################################################################
+
+#######################################################################
+#
+# perf_mariadb.sh
+# Description:
+#    Install mariadb and sysbench.
+#    Run performance test of MariaDB using sysbench.
+#    This script needs to be run on client VM.
+#
+# Supported Distros:
+#    Ubuntu/Suse/Centos/RedHat/Debian
+#######################################################################
+CONSTANTS_FILE="./constants.sh"
+UTIL_FILE="./utils.sh"
+HOMEDIR=$(pwd)
+LOG_FOLDER="${HOMEDIR}/mariadb_log"
+MARIADB_RESULT="${LOG_FOLDER}/report.csv"
+MARIADB_LOG_NAME="mariadb.bench.log"
+COMMON_LOG_FILE="${LOG_FOLDER}/common.log"
+SYSBENCH_VERSION=1.0.20
+
+db_path="/maria/db"
+db_parent_path="/maria"
+escaped_path=$(echo "${db_path}" | sed 's/\//\\\//g')
+user="lisa"
+
+. ${CONSTANTS_FILE} || {
+	LogErr "Error: missing ${CONSTANTS_FILE} file"
+	SetTestStateAborted
+	exit 0
+}
+
+. ${UTIL_FILE} || {
+	LogErr "Missing ${UTIL_FILE} file"
+	SetTestStateAborted
+	exit 0
+}
+
+if [ ! "${server}" ]; then
+	LogErr "Please add/provide value for server in constants.sh. server=<server ip>"
+	SetTestStateAborted
+	exit 0
+fi
+if [ ! "${client}" ]; then
+	LogErr "Please add/provide value for client in constants.sh. client=<client ip>"
+	SetTestStateAborted
+	exit 0
+fi
+
+if [ ! "${THREADS}" ]; then
+	THREADS=(1 2 4 8 16 32 64 128 256)
+fi
+
+if [ ! "${MAX_TIME}" ]; then
+	MAX_TIME=300
+fi
+
+function install_sysbench () {
+	LogMsg "Getting sysbench"
+	wget https://github.com/akopytov/sysbench/archive/${SYSBENCH_VERSION}.tar.gz
```

Comment:
I want to use the same version of sysbench in various distros. So the comparison test results will be more valuable.

Reply:
OK, that is fair reply, but we don't want to make a dependency of someone else's person repo. I don't think it hurts to use distro recommended version. They would verify the version when they add it to the repo. Right?

Reply:
Thank you for your comment. The version of sysbench in Ubuntu 18.04 is 1.0.11, in Centos is 1.0.17. I think if we use different version of sysbench to measure mariadb performance, it is difficult to figure out why the mariadb performance is poor or good, because we don't know if the different version of sysbench is one reason.

Reply:
@lubaihua33 I replied to your question over Teams also. I let you decide the choice between open source from the person repo or different sysbench version in distro's repo. You already understand pros and cons.

Reply:
I don't know if it is discussed before, but as it is someone else maintaining the repo should we keep a fork on our end and update that accordingly?

Reply:
@juhlee-microsoft I decide to keep the fixed release of the Sysbench.
@abhimarathe I think we don't want to maintain another code repo. I think using a fixed version release is good.